app/services/container_dashboard_service.rb

```diff
@@ -175,8 +173,54 @@ def ems_utilization
           :yData => ["used"] + used_mem.values.map { |m| (m / 1024.0).round }
         }
       }
-    else
-      {}
     end
   end
+
+  def hourly_network_trends
+    resource_ids = @ems.present? ? [@ems.id] : ManageIQ::Providers::ContainerManager.all.pluck(:id)
+    hourly_network_trend = Hash.new(0)
+    hourly_metrics =
+      MetricRollup.find_all_by_interval_and_time_range("hourly", 24.hours.ago.beginning_of_hour.utc, Time.now.utc)
+    hourly_metrics =
+      hourly_metrics.where('resource_type = ? AND resource_id in (?)', 'ExtManagementSystem', resource_ids)
```

Comment:
> Im not sure I understand. You meant to get these metrics without specifying ids? if so, maybe this could be a join+where with ems instead.

Cant get something like this to work due to the polymorphic association

I was hoping for something like this:
```Ruby
hourly_metrics.includes(:resource).where(
  :resource_type => 'ExtManagementSystem',
  :resource => {
    :type => ManageIQ::Providers::ContainerManager.types
} )
```